reporter/src/funTest/kotlin/reporters/AntennaAttributionDocumentReporterTest.kt

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Sorry, some more commit message nits:

- "reporter: Add *a* test ..."
- IMO you can drop "and uses for this the scanners test result" but in exchange say why we're not doing something more sophisticated than just looking at the file size. E.g. "The AntennaAttributionDocumentReporterTest tests only the size of the generated report because comparing the contents does not work due to added timestamps."